docs/lab/06/index.md

```diff
@@ -0,0 +1,359 @@
+---
+sidebar_position: 7
+description: Serial Peripheral Interface
+slug: /lab/06
+---
+
+# 06 - Serial Peripheral Interface
+
+This lab will teach you how to communicate with hardware devices using the Serial Peripheral Interface (SPI) protocol, in Embassy. 
+
+## Resources
+
+TBD
+
+## Serial Peripheral Interface (SPI)
+
+The Serial Peripheral Interface (or SPI) is a communication protocol used between microcontrollers and peripheral ICs (integrated circuits). It allows synchronous transfer of data between one *main* device and one or more *sub* devices (e.g. sensors). The main device is the one that dictates the communication.
+
+### Data transmission
+
+![SPI_protocol](images/spi_network.svg)
+
+During SPI transmission, there are 4 wires used:
```

Comment:
Not necessarily, elaborate about half-duplex, full-duplex and transmission without CS.

Reply:
I explained that there is also a 3 wire SPI configuration, that provides a half-duplex communication. Anything else I should add?